git/git_remotes.md

```diff
@@ -1,22 +1,70 @@
+
+Cloning is Git's way of getting a copy of a project. You'll have to clone a repository before you can make changes to it or see the source code on your own computer. Cloning helps you edit a repository on your local computer without making irreversible changes.
+
+Remember forking is specific to GitHub, and cloning is a feature of Git. 
+
+## Assignment
+1. Watch [episode 1.3](https://www.youtube.com/watch?v=_NrSWLQsDL4&list=PLRqwX-V7Uu6ZF9C0YMKuns9sLDzK6zoiV&index=3) of [The Coding Train](https://www.youtube.com/channel/UCvjgXvBlbQiydffZU7m1_aw)'s [Git and GitHub for Poets](https://www.youtube.com/playlist?list=PLRqwX-V7Uu6ZF9C0YMKuns9sLDzK6zoiV) to learn about making forks on GitHub
+2. Watch [episode 1.6](https://youtu.be/yXT1ElMEkW8?list=PLRqwX-V7Uu6ZF9C0YMKuns9sLDzK6zoiV) of [The Coding Train](https://www.youtube.com/channel/UCvjgXvBlbQiydffZU7m1_aw)'s [Git and GitHub for Poets](https://www.youtube.com/playlist?list=PLRqwX-V7Uu6ZF9C0YMKuns9sLDzK6zoiV) to understand cloning a remote repository
+
+# Using the git remote Command
+Here, we will discuss using the `git remote` command. First we will learn to add a new remote, then we will see how to view all remotes on a git repository. 
 
 ## Assignment
-The list of resources the user will go through to learn about the topic of this lesson. Have no more than 5 resources, ideally no more than three.
+1. Read [GitHub](https://www.github.com)'s guide for [Adding a remote](https://help.github.com/articles/adding-a-remote/)
+2. Read [Listing Remote Branches](http://gitready.com/intermediate/2009/02/13/list-remote-branches.html) from [Nick Quaranto](http://gitready.com/)
+3. Read [GitHub](https://www.github.com)'s guide for [Removing a remote](https://help.github.com/articles/removing-a-remote/)
+
+# Working with a Remote Repository
+Finally, we are going to look at making changes to a repository and making a pull request to a remote repository that you don't own.
+
+## Assignment
+1. Watch this [60-second video](https://www.youtube.com/watch?v=-uQHV9GOA0w) covering git pull and git push
+2. Take a look at [Pushing and Pulling](http://gitready.com/beginner/2009/01/21/pushing-and-pulling.html) from [Nick Quaranto](http://gitready.com/)
+3. Watch [Git & GitHub: Pull requests](https://www.youtube.com/watch?v=FQsBmnZvBdc) from [Codecourse](https://www.youtube.com/watch?v=FQsBmnZvBdc)
 
 ## Exercises
-A group of exercises (If Applicable) for the student to complete in relation to the topic taught in the lesson.
+* Follow [this tutorial]((https://www.digitalocean.com/community/tutorials/how-to-create-a-pull-request-on-github) from [DigitalOcean](https://www.digitalocean.com) to see forking, cloning, remotes, and pull requests in action
```

Comment:
looks like theres an extra bracket on the link `[this tutorial]((https://w...`

Reply:
Good catch!